nuget/AppCenter.nuspec

```diff
@@ -18,15 +18,15 @@
       <group targetFramework="uap10.0">
         <dependency id="Microsoft.NETCore.UniversalWindowsPlatform" version="5.2.6"/>
         <dependency id="Newtonsoft.Json" version="12.0.2"/>
-        <dependency id="sqlite-net-pcl" version="1.3.1"/>
+        <dependency id="SQLitePCLRaw.bundle_green" version="[1.1.14, 2.0.0)" />
       </group>
       <group targetFramework="net45">
         <dependency id="Newtonsoft.Json" version="12.0.2"/>
-        <dependency id="sqlite-net-pcl" version="1.3.1"/>
+        <dependency id="SQLitePCLRaw.bundle_green" version="[1.1.14, 2.0.0)" />
       </group>
       <group targetFramework="netcoreapp3.0">
         <dependency id="Newtonsoft.Json" version="12.0.2"/>
-        <dependency id="sqlite-net-pcl" version="1.3.1"/>
+        <dependency id="SQLitePCLRaw.bundle_green" version="[1.1.14, 2.0.0)" />
         <dependency id="System.Management" version="4.6.0"/>
         <dependency id="System.Configuration.ConfigurationManager" version="4.6.0"/>
```

Comment:
Don't we need https://www.nuget.org/packages/System.Net.Http/ on .NET core? Can we test with internal nugets in demo to check?

Reply:
I guess it's still referenced implicitly (dependency of dependency) via `NETStandard.Library`

Reply:
https://github.com/microsoft/appcenter-sdk-dotnet/pull/1246#discussion_r355437350